src/Root.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { PeoplePage } from './components/Pages/PeoplePage.1';
```

Comment:
The import path for `PeoplePage` seems incorrect. The file is named `PeoplePage.1`, which is unusual for a component file. Ensure that the file name is correct and matches the import statement.